src/sentry/integrations/pagerduty/notify_action.py

```diff
@@ -124,15 +124,16 @@ def get_integrations(self):
         return integrations
 
     def get_services(self):
+        organization = self.project.organization
         integrations = Integration.objects.filter(
-            provider="pagerduty",
-            organizations=self.project.organization,
-            status=ObjectStatus.VISIBLE,
+            provider="pagerduty", organizations=organization, status=ObjectStatus.VISIBLE
         )
         services = []
         for integration in integrations:
             service_list = PagerDutyService.objects.filter(
-                organization_integration_id__in=integration.organizationintegration_set.all()
```

Comment:
@MeredithAnya What does `organizationintegration_set` represent exactly?

Reply:
All the OrganizationIntegration objects associated with that integration, I believe its a given thing when using Django's many to many field